test/functional/feature_llmq_connections.py

```
@@ -46,24 +49,24 @@ def run_test(self):
         self.generate(self.nodes[0], 1, sync_fun=self.no_op)
         for mn in self.get_quorum_masternodes(q):
-            self.wait_for_mnauth(mn.node, 4)
+            self.wait_for_mnauth(mn.get_node(self), 4)
 
         self.log.info("mine a new quorum and verify that all members connect to each other")
         q = self.mine_quorum()
 
         self.log.info("checking that all MNs got probed")
         for mn in self.get_quorum_masternodes(q):
-            self.wait_until(lambda: self.get_mn_probe_count(mn.node, q, False) == 4)
+            self.wait_until(lambda: self.get_mn_probe_count(mn.get_node(self), q, False) == 4)
```

Ruff (0.11.9)

Function definition does not bind loop variable `mn`

(B023)